docs/GetStarted/getting_started_linux_cmake.md

```diff
@@ -50,6 +50,14 @@ We recommend Clang. GCC is not fully supported.
 $ sudo apt install clang
 ```
 
+### Install Python
+
+LLVM requires python and its libraries:
```

Comment:
I'm not sure if the explanation on why Python is required adds enough here, and it could also be overly specific (other dependencies could require Python). How about removing this line?

Reply:
SGTM.